dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -896,10 +895,9 @@
           }
         },
         "confidenceScore": {
-          "x-ms-client-name": "confidence",
           "type": "number",
           "format": "double",
-          "description": "Confidence score between 0 and 1 of the extracted relation."
+          "description": "Confidence score between 0 and 1 of the extracted relation, rounded to 2 decimal places."
```

Comment:
Is this actually true? At least in CLU, it most often more than 2 decimal places. It's probably not even worth mentioning. They get a double back and probably don't need to know more. They should parse it as a double so precision matters little.

Reply:
In our case, we round the decimal to 2 places and send it. However, I've reverted the description as well.